platform/jvm/capture-timber/build.gradle.kts

```diff
@@ -119,8 +113,3 @@ publishing {
         }
     }
 }
-
-// TODO(murki): Using this requires further setup in CI and local (e.g. signing entries in the local gradle.properties file)
-signing {
```

Comment:
you can also just comment this section out

Reply:
since this is the error
```
Execution failed for task ':capture-timber:signMavenPublication'.
> Task :capture-timber:signMavenPublication FAILED
> Cannot perform signing task ':capture-timber:signMavenPublication' because it has no configured signatory
```

and I haven't set up that part yet